treeherder/model/derived/jobs.py

```diff
@@ -1702,14 +1704,20 @@ def store_result_set_data(self, result_sets):
         # TODO: Confirm whether we need to do a lookup in this loop in the
         #   memcache to reduce query overhead
         for result in result_sets:
-
+            top_revision = result['revisions'][-1]
```

Comment:
I guess this might be cleaner as:
```
top_revision = result['revisions'][-1]['revision']
```
and then both the references to it in the `.append()` as well as the `short_top_revision` will be shorter?